src/styles/blocks/stars.scss

```
@@ -0,0 +1,22 @@
+.stars {
+  display: flex;
+
+  &__star {
+    display: inline-block;
```

why do you need `inline-block`? you already have flex around
```suggestion
    display: block;
```

---

done